test/response_to_json_translator_test.cc

```diff
@@ -249,7 +256,7 @@ class ResponseToJsonTranslatorTestCase {
 
 class ResponseToJsonTranslatorTest : public ::testing::Test {
  protected:
-  ResponseToJsonTranslatorTest() : streaming_(false) {}
+  ResponseToJsonTranslatorTest() : streaming_(false), json_response_translate_options_({ pbutil::JsonPrintOptions(), false }) {}
```

Comment:
the line is too long

Reply:
Sorry, the script `scripts/check-style` is suppose to do this, but it is too old now.  It needs to be fixed.

Reply:
I fixed "check-style" script in https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/pull/74.